Assets/Scripts/Player/Commands/ReloadCommand.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Player.Commands
+{
+    public class ReloadCommand : Command
+    {
+        public override void Execute(Controller controller)
+        {
+            // TODO: Should call a method similar to `controller.ActiveWeapon.Reload()`
```

Comment:
Just to be sure - this method will be implemented by Alessandro when he glues the HUD, weapons, and character movement together, right?

Reply:
Yes